owtf.rst

```diff
@@ -1,18 +1,17 @@
 Overview
 ========
 
-The purpose of this tool is to automate the manual, uncreative part of pen testing:
-For example, spending time trying to remember how to call "tool X", parsing results
-of "tool X" manually to feed "tool Y", etc.
+The purpose of this tool is to automate the manual and uncreative parts of pen testing.
+For example,
+Figuring out how to call "tool X" then parsing results of "tool X" manually to feed "tool Y" and so on is time consuming.
 
 By reducing this burden we hope pen testers will have more time to:
 
-* See the big picture and think out of the box
-* More efficiently find, verify and combine vulnerabilities
-* Have time to investigate complex vulnerabilities like business logic/architectural flaws or virtual hosting sessions
+* See the big picture and think out of the box,
+* Find, verify and combine vulnerabilities efficiently,
+* Investigate complex vulnerabilities like business logic, architectural flaws, virtual hosting sessions, etc.
```

Comment:
In this case, original is more correct. OWTF saves time and hence "have time" is relevant.